source/PerformanceTests/ExamplePerformanceTests/MultipleSailfishVariablesExample.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Sailfish.Attributes;
+using System;
+
+namespace PerformanceTests.ExamplePerformanceTests;
+
+[Sailfish]
+public class MultipleSailfishVariablesExample
+{
+    [SailfishVariable(1, 2, 3)]
+    public int? VarA { get; set; }
+
+    [SailfishVariable(2, 4, 6)]
+    public int? VarB { get; set; }
+
+    [SailfishVariable(2, 4, 6)]
+    public int? VarC { get; set; }
+
+    public string? VarD { get; set; }
+    public string? VarE { get; set; }
+
+    [SailfishGlobalSetup]
+    public void GlobalSetup()
+    {
+        VarD = "This is set";
```

Comment:
You might need to do something like call to upper on the strings, since these strings I think are constant and never collected. But if you to upper them they will be generated at runtime and so the result can be collected.